static/src/stylesheets/module/commercial/_creatives.scss

```diff
@@ -647,6 +647,36 @@
         display: block;
         clear: left;
     }
+
+    &.creative--ad-feature-mpu-large-v2 {
```

Comment:
Do we need to put it inside the `.creative--ad-feature-mpu-large`? It is adding unnecessary nesting/specificity.